metals/src/main/scala/scala/meta/internal/metals/CodeAction.scala

```diff
@@ -16,6 +16,8 @@ trait CodeAction {
    */
   def kind: String
 
+  def supportScala3: Boolean
```

Comment:
Maybe we should set the default value to true and override only in relevant actions?